src/components/booksForm/booksForm.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable arrow-body-style */
```

Comment:
## Move the book component
- [ ] The `bookForm.js` component needs to be moved to the `container` directory. You only need to have the `App.js` and the `book.js` components in this directory.